<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MDMForm</class>
 <widget class="QWidget" name="MDMForm">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基础数据配置</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>992</width>
     <height>536</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,6,0,0,3">
      <item>
       <widget class="QLabel" name="MDMlable">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>基础数据配置表</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="MDMListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>260</width>
          <height>350</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,9">
        <item>
         <widget class="QLabel" name="temFilel">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模板文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temFile">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模板文件</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9">
        <item>
         <widget class="QLabel" name="temSheetl">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模板表格</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="temSheet">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>模板表格</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,8">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,2">
            <item>
             <widget class="QLabel" name="temStartl">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>数据起始行</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="temStart">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>起始行</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="8,2">
            <item>
             <widget class="QLabel" name="temEndl">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>数据终止行</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="temEnd">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>终止行</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnUpdate">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>更新模板文件数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="btnTemplate">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>打开模板文档</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExport">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>下载当前数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnImport">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>上传模板数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>配置数据</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="dataTableWidget">
        <property name="minimumSize">
         <size>
          <width>720</width>
          <height>470</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>MDMListWidget</tabstop>
  <tabstop>btnTemplate</tabstop>
  <tabstop>btnExport</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>dataTableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
